src/profile-management/src/employees/employees.service.ts

```diff
@@ -1,44 +1,44 @@
 import { Injectable } from '@nestjs/common';
-import { EmployeesRepository } from './employees.repository';
-import { Employee } from '../shared/schemas/employee.schema';
+import { CreateEmployeeDto } from './dto/create-employee.dto';
 import { UpdateEmployeeDto } from './dto/update-employee.dto';
+import { EmployeesRepository } from './employees.repository';
+import { Employee } from './schemas/employee.schema';
 import { FilterQuery } from 'mongoose';
-import { hashPassword } from 'src/shared/helper';
-import { CreateEmployeeDto } from './dto/create-employee.dto';
+import { hashPassword } from 'src/employees/helper';
 
 @Injectable()
 export class EmployeesService {
   constructor(private readonly employeesRepository: EmployeesRepository) {}
 
-  async getEmployeeById(id: string): Promise<Employee> {
-    return this.employeesRepository.findOne({ _id: id });
-  }
-
-  async getEmployeeByPersonalId(personalId: string): Promise<Employee> {
-    return this.employeesRepository.findOne({ personalId: personalId });
+  async createEmployee(
+    createEmployeeDto: CreateEmployeeDto,
+  ): Promise<Employee> {
+    createEmployeeDto.hash = await hashPassword(createEmployeeDto.hash);
+    return this.employeesRepository.create({ createEmployeeDto });
   }
 
   async getEmployees(): Promise<Employee[]> {
     return this.employeesRepository.find({});
   }
 
-  async createEmployee(
-    createEmployeeDto: CreateEmployeeDto,
-  ): Promise<Employee> {
-    createEmployeeDto.hash = await hashPassword(createEmployeeDto.hash);
-    return this.employeesRepository.create({ createEmployeeDto });
+  async getEmployeeById(id: string): Promise<Employee> {
+    return this.employeesRepository.findOne({ _id: id });
```

Comment:
what you can do in a future issue, is to add a middleware to the findById request where it checks if the given`id` which comes from the front-end is indeed a realy mongoId, because its not a good practice to rely on the db to check if its correct